packages/flutter_test/lib/src/matchers.dart

```diff
@@ -221,6 +221,18 @@ const Matcher isInCard = _IsInCard();
 ///  * [isInCard], the opposite.
 const Matcher isNotInCard = _IsNotInCard();
 
+/// Asserts that the object represents the same color swatch as [color] when
+/// used to paint.
+///
+/// Specifically this matcher checks the object is of type [ColorSwatch] and its
+/// color components fall below the delta specified by [threshold].
+///
+/// Note: This doesn't recurse into the swatches [Color] type, instead treating
+/// them as [Color]s.
+Matcher isSameColorSwatchAs<T>(ColorSwatch<T> color, {double threshold = 0.004}) {
```

Comment:
Super-nit: `const colorEpsilon = 0.004;` somewhere and share between these two?

Reply:
done